app/models/ranking/api_client.rb

```diff
@@ -0,0 +1,66 @@
+require 'googleauth/stores/redis_token_store'
+require 'google/apis/analyticsreporting_v4'
+
+class Ranking::ApiClient
+
+  def initialize()
+    if use_api?
+      @analytics = Google::Apis::AnalyticsreportingV4
```

Comment:
Wow, it seems strange for me 🙈 
If you don't want to type `Google::Apis::AnalyticsreportingV4` always,
you can:

```ruby
class Ranking::ApiClient
  Analytics = Google::Apis::AnalyticsreportingV4

  def initialize
    Analytics::AnalyticsReportingService.new
...
```